libs/domain/cart/services/src/default-cart.service.ts

```diff
@@ -101,12 +102,20 @@ export class DefaultCartService implements CartService {
     },
   });
 
+  protected removeCouponCommand$ = createCommand({
+    ...this.cartCommandBase,
+    action: (qualifier: CouponQualifier) => {
+      return this.adapter.deleteCoupon(qualifier);
+    },
+    onSuccess: [...this.cartCommandBase.onSuccess, CartEntryRemoved],
```

Comment:
Probably query events are mixed up, this uses `CartEntryRemoved`, and entry command - `CouponRemoved`